podium-uaa/src/main/java/nl/thehyve/podium/service/OrganisationService.java

```diff
@@ -216,7 +216,20 @@ public OrganisationDTO activation(Long id, boolean activated) {
     }
 
     /**
-     * Mark the organisation as deleted.
+     *  Get one organisation by short name.
+     *
+     *  @param shortName the short name of the entity
+     *  @return the entity
+     */
+    @Transactional(readOnly = true)
+    public Organisation findByShortName(String shortName) {
```

Comment:
A specific method only used in [TestService](podium-uaa/src/main/java/nl/thehyve/podium/service/TestService.java)?